internal/tests/tx_check_test.go

```diff
@@ -14,6 +14,7 @@ import (
 
 func TestTx_RecursivelyCheckPages_MisplacedPage(t *testing.T) {
 	db := btesting.MustCreateDB(t)
+	db.StrictMode = false
```

Comment:
`db.forceDisableStrictMode()`

I think that this should be a method such that it can: 
 - log that the default was overriden. 
 - it's not easy to follow that we do 2-layers override of underlying bbolt state (i.e. btesting.DB expends bbolt.DB)

Reply:
thx & updated.